website/pages/en/publishing/publishing-a-subgraph.mdx

```diff
@@ -28,6 +28,16 @@ Developers can add GRT signal to their subgraphs. If a subgraph is eligible for
 
 > If your subgraph is eligible for rewards, is recommended that you curate your own subgraph with at least 3,000 GRT (as of April 11th, 2024) in order to attract additional indexers to index your subgraph
 
+The [Sunrise Upgrade Indexer](/sunrise/#what-is-the-upgrade-indexer) ensures the indexing of all subgraphs, signaling GRT on a particular subgraph will draw more indexers to it. This incentivization of additional Indexers through curation aims to enhance the quality of service for queries by reducing latency and enhancing network availability.
+
+When signaling, Curators can decide to signal on a specific version of the subgraph or to signal using auto-migrate. If they signal using auto-migrate, a curator’s shares will always be updated to the latest version published by the developer. If they decide to signal on a specific version instead, shares will always stay on this specific version.
+
+To assist teams that are transitioning subgraphs from the hosted service to the Graph Network, curation support is now available. If you require assistance with curation to enhance the quality of service, please send a request to the Edge & Node team at support@thegraph.zendesk.com and specify the subgraphs you need assistance with.
+
+Indexers can find subgraphs to index based on curation signals they see in Graph Explorer (screenshot below).
```

Comment:
```suggestion
Indexers can find subgraphs to index based on curation signals they see in Graph Explorer.
```
The screenshot is right after this paragraph, so it seems redundant to mention it. :P But maybe we could replace the `.` by a `:`?